app-modules/project/src/Filament/Resources/ProjectResource/Pages/ManageFiles.php

```diff
@@ -67,13 +66,6 @@ public static function getNavigationLabel(): string
         return 'Files';
     }
 
-    public static function canAccess(array $arguments = []): bool
-    {
-        $user = auth()->user();
-
-        return ProjectFileFeature::active() && $user->can('viewAny', [ProjectFile::class, $arguments['record']]);
-    }
```

Comment:
This `canAccess` actually does have a specific override in the `$user->can('viewAny', [ProjectFile::class, $arguments['record']])` that should be retained.

In fact, you can see that a test is failing here because this was removed.

Restore this override and keep the `$user->can('viewAny', [ProjectFile::class, $arguments['record']])` part.